test_regress/t/t_x_rand_mt_stability.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2024 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+import vltest_bootstrap
+import glob
+
+test.scenarios("vltmt")
```

Comment:
I had to create multi-threaded variants of the stability tests because the `$random()` results were different between them and the single-threaded variants.  I don't know if that's expected or not (seems plausible to me that it would matter).

The non-`$random()` results do appear stable between the single and multi-threaded runs, but I am not currently enforcing that.

Reply:
I wouldn't expect single to match multithreaded randomization, too complicated & slow to enforce, so fine.